### ISSUE TITLE
Creating an organization from an account fails with: data.rb:217:in `initialize': wrong number of arguments (given 1, expected 0) (ArgumentError)

### ISSUE THREAD

**Issue by dcermak** [hashicorp/vagrant_cloud#75]
I am trying to reproduce the following example from the readme:
```ruby
account = VagrantCloud::Account.new(access_token: "MY_TOKEN")
org = account.organization(name: "dcermak")
```
but unfortunately, that fails with the following error:
```
/home/dan/suse/github.com/dcermak/obs-to-vagrantcloud/vendor/ruby/3.0.0/gems/vagrant_cloud-3.0.4/lib/vagrant_cloud/data.rb:217:in `initialize': wrong number of arguments (given 1, expected 0) (ArgumentError)
        from /home/dan/suse/github.com/dcermak/obs-to-vagrantcloud/vendor/ruby/3.0.0/gems/vagrant_cloud-3.0.4/lib/vagrant_cloud/box.rb:21:in `initialize'
        from /home/dan/suse/github.com/dcermak/obs-to-vagrantcloud/vendor/ruby/3.0.0/gems/vagrant_cloud-3.0.4/lib/vagrant_cloud/data.rb:211:in `new'
        from /home/dan/suse/github.com/dcermak/obs-to-vagrantcloud/vendor/ruby/3.0.0/gems/vagrant_cloud-3.0.4/lib/vagrant_cloud/data.rb:211:in `load'
        from /home/dan/suse/github.com/dcermak/obs-to-vagrantcloud/vendor/ruby/3.0.0/gems/vagrant_cloud-3.0.4/lib/vagrant_cloud/organization.rb:16:in `block in initialize'
        from /home/dan/suse/github.com/dcermak/obs-to-vagrantcloud/vendor/ruby/3.0.0/gems/vagrant_cloud-3.0.4/lib/vagrant_cloud/organization.rb:14:in `map'
        from /home/dan/suse/github.com/dcermak/obs-to-vagrantcloud/vendor/ruby/3.0.0/gems/vagrant_cloud-3.0.4/lib/vagrant_cloud/organization.rb:14:in `initialize'
        from /home/dan/suse/github.com/dcermak/obs-to-vagrantcloud/vendor/ruby/3.0.0/gems/vagrant_cloud-3.0.4/lib/vagrant_cloud/data.rb:211:in `new'
        from /home/dan/suse/github.com/dcermak/obs-to-vagrantcloud/vendor/ruby/3.0.0/gems/vagrant_cloud-3.0.4/lib/vagrant_cloud/data.rb:211:in `load'
        from /home/dan/suse/github.com/dcermak/obs-to-vagrantcloud/vendor/ruby/3.0.0/gems/vagrant_cloud-3.0.4/lib/vagrant_cloud/account.rb:100:in `organization'
        from (irb):6:in `<main>'
        from /home/dan/suse/github.com/dcermak/obs-to-vagrantcloud/vendor/ruby/3.0.0/gems/irb-1.3.5/exe/irb:11:in `<top (required)>'
        from /home/dan/suse/github.com/dcermak/obs-to-vagrantcloud/vendor/ruby/3.0.0/bin/irb:23:in `load'
        from /home/dan/suse/github.com/dcermak/obs-to-vagrantcloud/vendor/ruby/3.0.0/bin/irb:23:in `<top (required)>'
        from /usr/share/gems/gems/bundler-2.2.15/lib/bundler/cli/exec.rb:63:in `load'
        from /usr/share/gems/gems/bundler-2.2.15/lib/bundler/cli/exec.rb:63:in `kernel_load'
        from /usr/share/gems/gems/bundler-2.2.15/lib/bundler/cli/exec.rb:28:in `run'
        ... 12 levels...
```

I am unfortunately pretty much lost here, what am I doing wrong?

**Comment by dcermak** [comment]
This appears to be related to ruby 3.0, because the same code works flawlessly with ruby 2.7

**Comment by antonakv** [comment]
```
$ rvm --default use 2.7
$ ruby -v
ruby 2.7.2p137 (2020-10-01 revision 5445e04352) [x86_64-darwin20]
$ vagrant cloud auth login                                                                                                          
You are already logged in.
$ vagrant cloud box create myuser/test64 
/opt/vagrant/embedded/gems/2.2.17/gems/vagrant_cloud-3.0.4/lib/vagrant_cloud/data.rb:217:in `initialize': wrong number of arguments (given 1, expected 0) (ArgumentError)
	from /opt/vagrant/embedded/gems/2.2.17/gems/vagrant_cloud-3.0.4/lib/vagrant_cloud/box.rb:21:in `initialize'
	from /opt/vagrant/embedded/gems/2.2.17/gems/vagrant_cloud-3.0.4/lib/vagrant_cloud/data.rb:211:in `new'
	from /opt/vagrant/embedded/gems/2.2.17/gems/vagrant_cloud-3.0.4/lib/vagrant_cloud/data.rb:211:in `load'
	from /opt/vagrant/embedded/gems/2.2.17/gems/vagrant_cloud-3.0.4/lib/vagrant_cloud/organization.rb:16:in `block in initialize'
	from /opt/vagrant/embedded/gems/2.2.17/gems/vagrant_cloud-3.0.4/lib/vagrant_cloud/organization.rb:14:in `map'
	from /opt/vagrant/embedded/gems/2.2.17/gems/vagrant_cloud-3.0.4/lib/vagrant_cloud/organization.rb:14:in `initialize'
	from /opt/vagrant/embedded/gems/2.2.17/gems/vagrant_cloud-3.0.4/lib/vagrant_cloud/data.rb:211:in `new'
	from /opt/vagrant/embedded/gems/2.2.17/gems/vagrant_cloud-3.0.4/lib/vagrant_cloud/data.rb:211:in `load'
	from /opt/vagrant/embedded/gems/2.2.17/gems/vagrant_cloud-3.0.4/lib/vagrant_cloud/account.rb:100:in `organization'
	from /opt/vagrant/embedded/gems/2.2.17/gems/vagrant-2.2.17/plugins/commands/cloud/box/create.rb:61:in `create_box'
	from /opt/vagrant/embedded/gems/2.2.17/gems/vagrant-2.2.17/plugins/commands/cloud/box/create.rb:43:in `execute'
	from /opt/vagrant/embedded/gems/2.2.17/gems/vagrant-2.2.17/plugins/commands/cloud/box/root.rb:46:in `execute'
	from /opt/vagrant/embedded/gems/2.2.17/gems/vagrant-2.2.17/plugins/commands/cloud/root.rb:73:in `execute'
	from /opt/vagrant/embedded/gems/2.2.17/gems/vagrant-2.2.17/lib/vagrant/cli.rb:67:in `execute'
	from /opt/vagrant/embedded/gems/2.2.17/gems/vagrant-2.2.17/lib/vagrant/environment.rb:290:in `cli'
	from /opt/vagrant/embedded/gems/2.2.17/gems/vagrant-2.2.17/bin/vagrant:231:in `<main>'
```
It doesn't work with 2.7 as well

**Comment by chrisroberts** [comment]
Fixed via #76. Cheers!
